test/runtime/fqdn.go

```diff
@@ -331,6 +332,64 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		res.ExpectSuccess("Cannot access to allowed DNS name %q", world1Target)
 	})
 
+	It("Validate dns-proxy monitor information", func() {
+		policy := `
+[
+	{
+		"labels": [{
+			"key": "monitor"
+		}],
+		"endpointSelector": {
+			"matchLabels": {
+				"container:id.app1": ""
+			}
+		},
+		"egress": [
+			{
+				"toPorts": [{
+					"ports":[{"port": "53", "protocol": "ANY"}],
+					"rules": {
+						"dns": [
+							{"matchPattern": "world1.cilium.test"}
+						]
+					}
+				}]
+			},
+			{
+				"toFQDNs": [{
+					"matchPattern": "world1.cilium.test"
+				}]
+			}
+		]
+	}
+]`
+		_, err := vm.PolicyRenderAndImport(policy)
+		Expect(err).To(BeNil(), "Policy cannot be imported")
+
+		expectFQDNSareApplied()
+
+		ctx, cancel := context.WithCancel(context.Background())
+		monitorCMD := vm.ExecInBackground(ctx, "cilium monitor")
+		defer cancel()
+
+		allowVerdict := "verdict Forwarded DNS Query: world1.cilium.test"
+		deniedVerdict := "verdict Denied DNS Query: world2.cilium.test"
```

Comment:
Should we use the json output instead? I'm not sure if that's as easy to check

Reply:
TBH, I think that it's clear if we use plaintext, if not match json is a bit challenging and it's hard to read the test.

Reply:
It is definitely more human readable. I guess we'll see how often it changes :P